server/webhook.go

```diff
@@ -544,15 +544,6 @@ func (s *Server) findProjectDatabases(ctx context.Context, projectID int, tenant
 		return nil, errors.Errorf("project %d does not have database %q", projectID, dbName)
 	}
 
-	// Tenant mode does not allow filtering databases by environment and expect
-	// multiple databases with the same name.
```

Comment:
@d-bytebase - What fact has changed that invalidates this?

Reply:
Probably some stale code here for no reason. The findProjectDatabases should only be used for non-tenant case.